src/main/java/com/basho/riak/client/api/commands/mapreduce/BinaryValueSerializer.java

```diff
@@ -0,0 +1,21 @@
+package com.basho.riak.client.api.commands.mapreduce;
+
+import com.basho.riak.client.core.util.BinaryValue;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+
+/**
+ * Serializes a {@link BinaryValue} to a UTF-8 string for the Riak MapReduce JSON format.
+ */
+public class BinaryValueSerializer extends JsonSerializer<BinaryValue>
```

Comment:
Should it be registered somehow to be used by default without a necessity to be registered explicitly by user?